superbench/common/utils/topo_aware.py

```diff
@@ -31,6 +36,34 @@ def search(self, pattern, string, flags=0):
         return self.matched
 
 
+def gen_ibstat_file(ibstat_file):
+    """Generate ibstat file for each node with specified path.
+
+    Args:
+        ibstat_file (str): path of ibstat output.
+    """
+    comm = MPI.COMM_WORLD
+    name = MPI.Get_processor_name()
+
+    # The command to fetch ibstat info
+    cmd = r"ibstat | grep -Po 'System image GUID: \K\S+$'"
```

Comment:
What's letter 'r' for?

Reply:
For the lint test
`
./superbench/common/utils/topo_aware.py:49:50: W605 invalid escape sequence '\K'
./superbench/common/utils/topo_aware.py:49:52: W605 invalid escape sequence '\S'
`